test/bitgo/keyutil.js

```diff
@@ -0,0 +1,73 @@
+/* global describe, it */
+
+const assert = require('assert')
+var randombytes = require('randombytes')
+
+const BigInteger = require('bigi')
+
+const {
+  bitgo: {
+    keyutil: {
+      privateKeyBufferToECPair,
+      privateKeyBufferFromECPair
+    }
+  },
+  ECPair
+} = require('../../src')
+
+describe('privateKeyBufferFromECPair', function () {
+  it('pads short private keys', function () {
+    var keyPair = new ECPair(BigInteger.ONE)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).byteLength, 32)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).toString('hex'),
+      '0000000000000000000000000000000000000000000000000000000000000001')
+  })
+
+  it('does not pad 32 bytes private keys', function () {
+    var hexString = 'a000000000000000000000000000000000000000000000000000000000000000'
+    var keyPair = new ECPair(new BigInteger(hexString, 16))
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).byteLength, 32)
+    assert.strictEqual(privateKeyBufferFromECPair(keyPair).toString('hex'), hexString)
+  })
+
+  it('throws if the key is too long', function () {
+    var hexString = '10000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
It's hard to see that this is one character longer, perhaps you could create a valid String and add a single character or something?